src/llv8.cc

```diff
@@ -672,6 +686,88 @@ std::string Smi::ToString(Error& err) {
 }
 
 
+/* Utility function to generate short type names for objects.
+ */
+std::string HeapObject::GetTypeName(InspectOptions* options, Error& err) {
+  int64_t type = GetType(err);
+  if (type == v8()->types()->kGlobalObjectType) return Inspect(options, err);
```

Comment:
I've gone with (Global), (Code) and (Map) to match the capitalisation on the other names. Those in turn should match what users see on heap dumps but let me know if you'd rather I used lower case.